en/config_fw/position_tuning_guide_fixedwing.md

```diff
@@ -1,13 +1,8 @@
-# Advanced Fixed-wing Position Tuning
+# Fixed-wing Position Tuning
 
 This guide offers some help in tuning the high-level fixed-wing controllers needed for flying missions and in altitude/position control mode.
 PX4 uses TECS for altitude and airspeed control, and L1 for horizontal heading/position control.
```

Comment:
Do we still use L1?

Reply:
No, I've replaced it with NPFG. 

We should explain better what NPFG does, but this is for me doesn't have to be with this PR, especially if we shuffle the docs around quite heavily soon.